src/libs/actions/OnyxUpdates.ts

```diff
@@ -1,12 +1,7 @@
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 
-/**
- *
- * @param {Number} [lastUpdateID]
- * @param {Number} [previousUpdateID]
- */
-function saveUpdateIDs(lastUpdateID = 0, previousUpdateID = 0) {
+function saveUpdateIDs(lastUpdateID = 0, previousUpdateID = 0): void {
```

Comment:
```suggestion
function saveUpdateIDs(lastUpdateID = 0, previousUpdateID = 0) {
```

I think we don't need to type functions as `void` when they don't return anything